jest.config.js

```diff
@@ -1,5 +1,9 @@
+/**
+ * @typedef {import('ts-jest')}
+ * @type {import('@jest/types').Config.InitialOptions}
+ */
 module.exports = {
-  coveragePathIgnorePatterns: ['/node_modules/'],
+  coveragePathIgnorePatterns: ['/node_modules/', '<rootDir>/src/util.ts'],
```

Comment:
I don't see why any file in src should be excluded from coverage. What's the reason for it?

Reply:
because I cannot really tests the branch: 
https://github.com/ridedott/eslint-plugin/blob/95af2249f8889a1293e4f01fd6ae0759f129e85e/src/util.ts#L13-L15
So coverage decreases. Any suggestions?
I could just disable coverage for that branch and not the entire file or revert back to how it was before without this branch.

Reply:
Why can't you test it? Do you have problems mocking the `require`?

Reply:
fixed with: https://github.com/ridedott/eslint-plugin/pull/30/commits/28da13ef1035388bb686a92daaf64053cfe39b1d